game/scripts/vscripts/Sections/Chapter6/SectionClosing.ts

```diff
@@ -176,13 +188,67 @@ function onStart(complete: () => void) {
         waitNpcsSpawned(),
 
         // Main logic
-        tg.fork([
+        tg.forkAny([
             // Play dialog
             tg.seq([
                 tg.audioDialog(LocalizationKey.Script_6_Closing_1, LocalizationKey.Script_6_Closing_1, slacks),
                 tg.audioDialog(LocalizationKey.Script_6_Closing_2, LocalizationKey.Script_6_Closing_2, sunsFan),
                 tg.audioDialog(LocalizationKey.Script_6_Closing_3, LocalizationKey.Script_6_Closing_3, slacks),
                 tg.audioDialog(LocalizationKey.Script_6_Closing_4, LocalizationKey.Script_6_Closing_4, sunsFan),
+
+                tg.withHighlights(
+                    tg.forkAny([
+                        tg.seq([
+                            tg.immediate(_ => {
+                                goalDestroyTowers.start()
+                                goalDestroyTowers.setValue(0)
+                                pathParticleID = createPathParticle([...pathLocations, ancient.GetAbsOrigin()])
+                            }),
+                            tg.completeOnCheck(_ => {
+                                towersToDestroy = towersToDestroy.filter(tower => unitIsValidAndAlive(tower))
+                                goalDestroyTowers.setValue(2 - towersToDestroy.length)
+
+                                return towersToDestroy.length === 0
+                            }, 0.1)
+                        ]),
+
+                        tg.seq([
+                            tg.panCameraExponential(getPlayerCameraLocation(), ancient.GetAbsOrigin(), 2),
+                            tg.audioDialog(LocalizationKey.Script_6_Closing_4, LocalizationKey.Script_6_Closing_4, sunsFan), // This needs to be edited with the new line for Sunsfan for destroying the ancient and the towers protecting it
+                            tg.panCameraExponential(ancient.GetAbsOrigin(), playerHero.GetAbsOrigin(), 2),
+                            tg.audioDialog(LocalizationKey.Script_6_Closing_4, LocalizationKey.Script_6_Closing_4, sunsFan), // This needs to be edited with the new line for Slacks for getting a Divine Rapier, and to destroy the ancient
+                            tg.immediate(() => playerHero.AddItemByName("item_rapier")),
```

Comment:
(If you want to) put this in the immediate below since it's already multiline anyway

Reply:
Good call, done.